### PR TITLE
fix - replace NaN values for all fields to None

### PR DESCRIPTION
**Pull request by viktorkertesz**
Dear @jiujing ,
Your latest change was a breaking one for me as I was relying on the old behavior where NaN values got into the inventory. Now I'd like to complete the new behavior for all values. In my understanding the only mandatory values in Nornir inventory are the name and hostname. All others are optional as those can come from groups and defaults.
I added this behavior for hostname too because on this level we should let Nornir to error out on wrong input. I say that a bit biased because I rely on the fact that name may be empty when I deal with switch stacks and the name field is empty for me. That's kinda sign for the code to skip those hosts. I incorporated that logic here so rows with empty "name" will be ignored which is fine as those would mix up anyway.

I added helper function _empty() to check variable more reliably for NaN value. str(var)=="nan" can give us false value when user tries to use "nan" as string.

### CODE REVIEW

**Comment by jiujing** [comment]
> 

Thank you for your PR.I will check as soon as I can.

File: nornir_table_inventory/plugins/inventory/table.py [hunk 1]
```diff
@@ -134,7 +140,8 @@ def load(self) -> Inventory:
         hosts = Hosts()
 
         for host_dict in self.hosts_list:
-            hosts[host_dict['name']] = _get_host_obj(host_dict)
+            if not _empty(host_dict['name']):
```
Comment (jiujing) [inline nornir_table_inventory/plugins/inventory/table.py:143]: In Nornir ，name must not be None. I think ,we should follow the raw rule. I know your thinking，this is for switch stacks or chasis switch such as Cisco nexus 2000. But I think ，Automation think over NOS（Network Operate System） more than Physical Switch. In our CMDB ,they are two model. I get data from NOS model., and parse data from NOS to NOS and physical switch.
And in our DC , every swith with a chasis must have a name.

Comment (viktorkertesz) [reply nornir_table_inventory/plugins/inventory/table.py:143]: That's the point! This part of the code will ignore rows with an empty name. But it still continues to parse others.
My original work on this was that you made it mandatory to fill in port field which is definitely not mandatory by Nornir. NaN values caused problem by converting float('nan') to int. So I made as many fields optional as possible.

**Comment by jiujing** [comment]
> Dear @jiujing , Your latest change was a breaking one for me as I was relying on the old behavior where NaN values got into the inventory. Now I'd like to complete the new behavior for all values. In my understanding the only mandatory values in Nornir inventory are the name and hostname. All others are optional as those can come from groups and defaults. I added this behavior for hostname too because on this level we should let Nornir to error out on wrong input. I say that a bit biased because I rely on the fact that name may be empty when I deal with switch stacks and the name field is empty for me. That's kinda sign for the code to skip those hosts. I incorporated that logic here so rows with empty "name" will be ignored which is fine as those would mix up anyway.
> 
> I added helper function _empty() to check variable more reliably for NaN value. str(var)=="nan" can give us false value when user tries to use "nan" as string.

you can upgrade by command " pip install nornir-table-inventory==0.4.4". Thank you for your PR!

**Pull request merged**